### PR TITLE
Adding HPK sensor geometry

### PR DESCRIPTION
**Pull request by irenedutta23**

### CODE REVIEW

**Comment by irenedutta23** [comment]
Initial geometry file for HPK pad sensor. Later PRs will include fixes for other parameters in the geometry file.

File: src/InitialAnalyzer.cc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #include <TFile.h>
 #include <iostream>
 #include <fstream>
-
+using namespace std;
```
Comment (cmadrid1) [inline src/InitialAnalyzer.cc]: Please remove this change. It is bad practice to do this in general but you have it as a global.

File: test/MyAnalysis.C [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include <functional>
 #include <unistd.h>
 
+using namespace std;
```
Comment (cmadrid1) [inline test/MyAnalysis.C]: Please remove this

File: test/MyAnalysis.C [hunk 3]
```diff
@@ -177,7 +178,7 @@ int main(int argc, char *argv[])
         {
             if(pair.first==analyzer) 
             {
-                std::cout<<"Running the " << analyzer << " Analyzer" <<std::endl;
+                std::cout<<"RunnIng the " << analyzer << " Analyzer" <<std::endl;
```
Comment (cmadrid1) [inline test/MyAnalysis.C]: Why did you capitalize the "I"? Just a strange change.

**Review by cmadrid1 — CHANGES_REQUESTED** [review]

**Review by cmadrid1 — APPROVED** [review]

**Pull request merged**